Gemfile.lock

```diff
@@ -61,6 +61,7 @@ GEM
     unicode-display_width (1.8.0)
 
 PLATFORMS
+  x86_64-darwin-17
```

Comment:
I don't remember Ruby very well, and am not sure why this was necessary.  I was asked to run `bundle lock --add-platform x86_64-darwin-17`, so I did.